tests/python/test-1-vector.py

```diff
@@ -198,3 +212,19 @@ def test_108(ceed_resource, capsys):
     assert abs(norm - 9.) < 1e-14
 
 # -------------------------------------------------------------------------------
+# Test modification of reshaped array
+# -------------------------------------------------------------------------------
+
+
+def test_199(ceed_resource):
```

Comment:
Was this supposed to be following the sequential local numbering here, i.e., `109` and it has a typo, or you decided that it won't follow the existing format because it does not reflect a C test?

Reply:
Since this does not correspond to a C test and it was numbered the same as the new test that does correspond to a C test, I renamed it.

Reply:
Yeah that's what I thought. Makes sense, thanks